recipes/conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "4.1.5" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/conda/conda
+  git_tag: {{ version }}
+  git_depth: 1
+
+build:
+  number: 0
+  script:
+    - unlink "${PREFIX}"/bin/conda                  # [unix]
```

Comment:
This is needed otherwise one messes up their `conda`.

```
$ conda build conda
-bash: /zopt/conda2/envs/nanshenv/bin/conda: /opt/anaconda1anaconda2anaconda3/bin/python: bad interpreter: No such file or directory
```